packages/did-resolver/src/resolver.ts

```diff
@@ -17,15 +25,59 @@ export class DIDResolverPlugin implements IAgentPlugin {
     if (!options.resolver) throw Error('Missing resolver')
     this.didResolver = options.resolver
 
-    this.resolveDid = this.resolveDid.bind(this)
-
     this.methods = {
-      resolveDid: this.resolveDid,
+      resolveDid: this.resolveDid.bind(this),
+      dereferenceDidUri: this.dereferenceDidUri.bind(this),
     }
   }
 
-  async resolveDid({ didUrl, options }: { didUrl: string, options?: DIDResolutionOptions }): Promise<DIDResolutionResult> {
+  /** {@inheritDoc @veramo/core#IResolver.resolveDid} */
+  async resolveDid({
+    didUrl,
+    options,
+  }: {
+    didUrl: string
+    options?: DIDResolutionOptions
+  }): Promise<DIDResolutionResult> {
     debug('Resolving %s', didUrl)
     return this.didResolver.resolve(didUrl, options)
   }
+
+  /** {@inheritDoc @veramo/core#IResolver.dereferenceDidUri} */
+  async dereferenceDidUri({
+    didDocument,
+    didURI,
+    section,
+  }: {
+    didDocument: DIDDocument
+    didURI: string
```

Comment:
```suggestion
    didUrl: string
```

Reply:
i also sometimes DID URI but i think the correct one is DID URL.

Reply:
got it.
It's fixed now